src/transport/UnauthenticatedSessionTable.h

```diff
@@ -82,7 +82,8 @@ class UnauthenticatedSession : public Session, public ReferenceCounted<Unauthent
     void Retain() override { ReferenceCounted<UnauthenticatedSession, UnauthenticatedSessionDeleter, 0>::Retain(); }
     void Release() override { ReferenceCounted<UnauthenticatedSession, UnauthenticatedSessionDeleter, 0>::Release(); }
 
-    ScopedNodeId GetPeer() const override { return ScopedNodeId(kUndefinedNodeId, GetFabricIndex()); }
+    ScopedNodeId GetPeer() const override { return ScopedNodeId(GetPeerNodeId(), kUndefinedFabricIndex); }
+    ScopedNodeId GetLocalScopedNodeId() const override { return ScopedNodeId(kUndefinedFabricIndex, kUndefinedFabricIndex); }
```

Comment:
That first arg to the ScopedNodeId constructor should be a node id, not a fabric index.

Reply:
Good catch.